pandas/core/generic.py

```diff
@@ -1906,6 +1906,10 @@ def _slice(self, slobj, axis=0, kind=None):
         return result
 
     def _set_item(self, key, value):
```

Comment:
this point is *only* for ``[]`` setting, e.g. setting a column on a DF or an element on a Series. ``.loc/.iloc`` are handled in core/indexing.py